azurerm/resource_arm_container_group_test.go

```diff
@@ -217,6 +217,7 @@ func TestAccAzureRMContainerGroup_linuxComplete(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "container.0.volume.0.read_only", "false"),
 					resource.TestCheckResourceAttr(resourceName, "os_type", "Linux"),
 					resource.TestCheckResourceAttr(resourceName, "restart_policy", "OnFailure"),
+					resource.TestCheckResourceAttr(resourceName, "log_analytics.#", "1"),
 				),
```

Comment:
Could we check some more values here? metadata/workspace_id etc

Reply:
It was `set`, that's why i am skipping it. Now, it's changed to list, i will add more validations.